providers/softlayer/src/main/java/org/jclouds/softlayer/features/VirtualGuestApi.java

```diff
@@ -149,4 +152,34 @@
    @Produces(MediaType.APPLICATION_JSON)
    @Fallback(Fallbacks.FalseOnNotFoundOr404.class)
    boolean setTags(@PathParam("id") long id, @BinderParam(TagToJson.class) Set<String> tags);
+
+   /**
+    * Set notes (visible in UI)
+    *
+    * @param id
+    *           id of the virtual guest
+    */
+   @Named("VirtualGuest:setNotes")
+   @POST
+   @Path("/SoftLayer_Virtual_Guest/{id}/editObject")
+   @Produces(MediaType.APPLICATION_JSON)
+   @Fallback(Fallbacks.FalseOnNotFoundOr404.class)
```

Comment:
404 fallbacks should not be used in write operations such as PUT or POST. Remove it.